src/components/Typography/Typography.types.ts

```diff
@@ -0,0 +1,76 @@
+import { TooltipProps } from 'antd'
+
+/**
+ * @link https://ant.design/components/typography#copyable
+ */
+export type CopyConfig = {
+
+  /**
+   * The text to copy.
+   */
+  text?: string;
+
+  /**
+   * Function called when the text is copied.
+   */
+  onCopy?: (event?: React.MouseEvent<HTMLDivElement>) => void;
```

Comment:
Have you tried to valorize this function? This type usually raises an error if a function without parameters is passed.

Reply:
I replaced it with our usual `() => void`